dev/tasks/exec/functions/git-commit.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+ 'use strict';
+
+const git = require( '../utils/git' );
+const PassThrough = require( 'stream' ).PassThrough;
+
+/**
+ * Adds only modified files to git repository and commits them with provided message
+ *
+ * @param {String} workdir
+ * @param {Object} params
+ * @returns {Object} stream
```

Comment:
I can't mix streamable and non-streamable functions in current task runner. Synchronous functions must return dummy stream. Otherwise Gulp won't notice that we finished the task.